src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Drawable {
+
+    void draw();
+
```

Comment:
```suggestion
    void draw();
```
pay attention to empty lines